tests/algorithms/test_required_on_load.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2022 MosaicML Composer authors
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import Type
+
+import pytest
+
+from composer import algorithms
+from composer.core import Algorithm, Time, TimeUnit  # type: ignore imports used in `eval(representation)`
+
+
+def initialize_algorithm(algo_cls: Type):
```

Comment:
We do have `algorithm_settings` that hold the minimal set of required `kwargs` if that helps